website/docs/interactors/3-locators-filters-actions.md

```diff
@@ -52,68 +50,81 @@ TextField('Username:', { id: 'username-id' }).exists();
 Filters can be very convenient for finding matching UI elements, but where they really shine is in making assertions about what you expect your application to be showing.
 
-In the [quick start](/docs/interactors/#making-test-assertions) we briefly touched on the assertion methods that are available on all interactors, `exists()` and `absent()`. There is also `has()` which allows you pass in a filter as its argument.
+In the [Quick Start](/docs/interactors/#making-test-assertions) we briefly touched on the assertion methods that are available for all interactors - `exists()` and `absent()`. There is also `has()` which allows you pass in a filter as its argument.
 
 :::note
- These assertion methods are equivalents of Jest's `expect` and Cypress' `should`. When refactoring your test with Interactors, you would replace those constructs with the interactors' assertion methods.
+ These different assertion methods are equivalents of Jest's `expect` and Cypress' `should`. When refactoring your test with Interactors, you would replace those constructs with the interactors' assertion methods.
 :::
 
-Continuing from the last example, this is how we would assert the placeholder against a textfield:
+In the case of a form with textfields, you would assert the placeholder against a textfield like this:
 
 ```js
 TextField({ id: 'username-id' }).has({ placeholder: 'USERNAME' });
 ```
 
-The difference between this approach and using `exists()` is that `exists()` is less explicit as it will succeed as long as there is at least one match. With the two textfields for the username and password, if we were to write a test like this:
+The difference between this approach and using `exists()` is that `exists()` is less refined as it will succeed as long as there is at least one match. With the two textfields for the username and password, the assertion would fail on if we were to write a test like this:
 
 ```js
 TextField().has({ placeholder: 'USERNAME' });
 ```
 
-This assertion would fail on account of the textfield that has the placeholder value `PASSWORD`. So you will need to choose the assertion method that is most appropriate for your tests.
+It would fail on account of the textfield that has the placeholder value `PASSWORD`. You therefore need to choose the assertion method that is most appropriate for your tests.
+
+Lastly, there is also the `is()` method, which functions in an almost identical way to `has()`. The only difference is in the semantics so that your tests can read better.
 
-Lastly, there is also the `is()` method which is identical to `has()`. The difference is only in semantics so that your tests can read better. For instance, if we wanted to test if a textfield is visible, `has()` would work perfectly fine but we would write the test using `is()` instead:
+For instance, if we wanted to test if a textfield is visible, `has()` would work perfectly fine; but writing the test using `is()` would look slightly differently like this:
```

Comment:
I think in this context maybe we should point out that it looks more "natural"?

Reply:
I think I just realized that we provide `is()` so that boolean filters can read more naturally. Can anybody think of a non-boolean filter that reads better with `is()` I'm drawing a blank:

```js
TextField().is({ disabled: true });
TextField().is({ visible: false });
TextField().is({ valid: true });

TextField().has({ placeholder: 'address' });
```

If this is the case, then we should explicitly mention this so that folks know the rule.

Reply:
Isn't `is` inherently a boolean question?

Reply:
But I'm worried saying `is() is for boolean assertions and has() for the rest` will make people think the functionality is different. So maybe we can phrase it so it's more of a suggestion?

Reply:
@jorgelainfiesta @cowboyd